shed/event_streams.py

```diff
@@ -231,37 +232,58 @@ def update(self, x, who=None):
         """
         return self.emit(self.dispatch(x))
 
-    def curate_streams(self, nds):
-        """Standardize name document pairs
+    def curate_streams(self, nds, outbound):
+        """Standardize inbound/outbound name document pairs
 
         Parameters
         ----------
         nds: tuple, or tuple of tuples
             The name document pair(s)
+        outbound: bool
+            If True curate streams outbound
 
         Returns
         -------
         name: str
             The name of the output doc(s)
-        docs: tuple
+        docs: {tuple, dict}
```

Comment:
is that correct [numpydoc syntax](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)? haven't seen it before. I think this works for values but not type?